docs/index.md

```diff
@@ -89,34 +89,71 @@ use-cases.
 
 
 
-
 ## Application Domains
 
-Native features of CrateDB paired with auxiliary software components provide
-optimal coverage of different application domains.
+Learn how to apply CrateDB's features to optimally cover use-cases in different
+application and topic domains.
 
 ::::{grid} 3
 :padding: 0
 
+:::{grid-item-card} Time Series Data
+:link: timeseries
+:link-type: ref
+:link-alt: Managing Time Series Data with CrateDB
+:padding: 3
+:text-align: center
+:class-card: sd-pt-3
+:class-body: sd-fs-1
+:class-title: sd-fs-5
+
+{material-outlined}`query_stats;1.3em`
+:::
```

Comment:
Currenty, "Time Series Data" and "Metrics" use the same icon. That detail may want to be improved by using a different one.

On [clients-tools » Integrations](https://cratedb.com/docs/crate/clients-tools/en/latest/#id1), it is used for "System Metrics" already, so this one should be replaced by [selecting another one](https://fonts.google.com/icons).

Reply:
Per https://fonts.google.com/icons?icon.query=measure, we've selected the [Stacked Line Chart icon](https://fonts.gstatic.com/s/i/short-term/release/materialsymbolsoutlined/stacked_line_chart/default/48px.svg). It has been implemented with 51f2788e4f8. Please let us know if you would like to select a different icon for that application domain item.

![image](https://github.com/crate/cratedb-guide/assets/453543/5c5dc6a2-b514-41a9-adc2-064ecd9b0253)